specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -10960,6 +11851,10 @@
           "$ref": "#/definitions/StorageProfile",
           "description": "Specifies the storage settings for the virtual machine disks."
         },
+        "capacityReservation": {
+          "$ref": "#/definitions/CapacityReservationProfile",
+          "description": "Specifies the capacity reservation related details of a virtual machine. <br><br>Minimum api-version: 2021-07-01."
```

Comment:
Specifies information about the capacity reservation that is used to allocate virtual machine. <br><br>Minimum api-version: 2021-07-01."

Reply:
updated